test/integration/auth.spec.ts

```diff
@@ -314,20 +314,66 @@ describe('admin.auth', () => {
   });
 
   it('updateUser() updates the user record with the given parameters', () => {
+    // TODO: replace all "any" types with explicit mfa related types after they are defined in d.ts file.
```

Comment:
Any reason for not updating the d.ts file in this PR? We can add the documentation comments in a future PR, but add the typings here.

Reply:
Yeah I delayed that because of the documentation comments. I have added the types. Will defer the comments for a later PR.